forum/qa-theme/SnowFlat/qa-styles.css

```diff
@@ -2271,7 +2271,7 @@ input[type="submit"], button {
 	color: #fff;
 }
 .qa-suggest-next a {
-	color: #fff;
+	color: #5faee3 !important;

```

Comment:
Ten important purystom CSS mógłby się nie spodobać.

Reply:
Hej, czy możesz mnie nakierować, które linki w stopce są niebieskie? Ja widzę białe: https://imgur.com/a/NxMPi
Co do użycia `!important` - przecież bez niego kolor się zmienia na niebieski, więc dlaczego go wstawiłeś? :) Chyba, że jakiś inny link nie chce zmienić koloru bez `!important`, to pokaż który.

Reply:
U mnie z kolei tak https://zapodaj.net/8ab7260f4e53a.png.html . Na jasnym jest ok, ale na ciemnym coś nie działa.

Reply:
Link po kliknięciu przechodzi w stan **visited** i jest ostylowany na niebiesko. Linki, w które nie kliknąłeś są koloru białego.

Reply:
Faktycznie. Ale i tak wg mnie powinno być jednym kolorem - brzydko wygląda :)

Reply:
Inny kolor jest właśnie po to, aby móc rozróżnić, czy link był już kliknięty czy nie.
Do poczytania: https://developer.mozilla.org/en-US/docs/Learn/CSS/Styling_text/Styling_links